src/main/java/com/basho/riak/client/core/query/crdt/types/RiakMap.java

```diff
@@ -237,6 +245,30 @@ public RiakRegister getRegister(String key)
         return unmodifiableMap(entries);
     }
 
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o)
+        {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass())
```

Comment:
Since this class is not marked as a 'final' it doesn't work for whatever descendant, IMO better to make all CRDT classes final.

Also not sure whether this default 'equals' implementation will be useful in the real life